servers/src/main/java/tachyon/master/MasterBase.java

```diff
@@ -155,6 +166,16 @@ public void stop() throws IOException {
         mStandbyJournalTailer.shutdownAndJoin();
       }
     }
+    mExecutorService.shutdownNow();
```

Comment:
This changes how masters are stopped and started now. Before, the threadpool was created only once, and never shutdown, even if the master was stopped and started. In the new implementation, the threadpool is shutdown in stop(), so a master can only be stopped once? Multiple starts and stops should be supported, since in TachyonMasterFaultTolerant, masters are only constructed once, and stopped and started repeatedly.